03/app.js

```diff
@@ -2,4 +2,28 @@ document.addEventListener('DOMContentLoaded', init);
 
 function init() {
     console.log('DOM');
+    getObjectsFromDocument();
+}
+
+function getObjectsFromDocument() {
+    const btn = document.querySelector('button');
+    btn.addEventListener('click', getIpNumber);
+    console.log('ok');
+}
+
+function getIpNumber() {
+    const api = fetch('https://www.ipify.org');
```

Comment:
Czy to zadanie Ci działało?! Tutaj jest błędny adres (poprawny: https://api.ipify.org?format=json) - zawsze patrz do dokumentacji! :)

Reply:
Mój błąd :) ctrl + z